Dockerfile

```diff
@@ -1,17 +1,22 @@
-FROM golang:1.6-alpine
+FROM golang:1.9-alpine
 
-RUN apk add --no-cache gcc git libc-dev
+RUN apk add --no-cache gcc git libc-dev curl
 
-ENV DOCKER_MACHINE_VERSION v0.7.0
+ENV DOCKER_MACHINE_VERSION v0.12.2
+RUN git clone --depth 1 -b "$DOCKER_MACHINE_VERSION" https://github.com/docker/machine.git "$GOPATH/src/github.com/docker/machine" \
+    && go build -v -o "$GOPATH/bin/docker-machine" github.com/docker/machine/cmd
```

Comment:
This is coming in via `dep` now, isn't it?  Do we still need to pull it in manually this way?

Reply:
Nice catch, I'll make the change.